### DISCUSSION TITLE
About set linkage h_szn_prm(h,szn)  in ReEDS

### DISCUSSION THREAD

**Discussion by narata** [NREL/ReEDS-2.0#71] [Q&A: Analysis] [answered]
Hi, ReEDS Team!

The reserve of non-dispatchable hydro can only comes into effect in timeslice in h3, h7, h11, h15. Why not other timeslice?

Many thanks!

The followings are related set and equation. 

set h_szn_prm(h,szn) "hour to season linkage for nd_hydro in the planning reserve margin constraint"
/
h3.summ
h7.fall
h11.wint
h15.spri
/ ;

eq_reserve_margin(r,szn,t)..      ..+ sum{(i,v,h)$[hydro_nd(i)$valgen(i,v,r,t)$h_szn_prm(h,szn)], GEN(i,v,r,h,t) }..

**Comment by wesleyjcole** [comment, marked as answer]
We make this simplification because we don't have robust profiles for nondispatchable hydropower.  By doing this, we are saying that the seasonal capacity credit of non-dispatchable hydropower is equal to capacity factor during the h3/h7/h11/h15 timeslice.  Happy to clarify if that doesn't make sense.

**Reply by narata** [reply]
Thanks for your clear answer!
